src/commonTest/kotlin/fr/acinq/lightning/payment/IncomingPaymentHandlerTestsCommon.kt

```diff
@@ -209,23 +224,14 @@ class IncomingPaymentHandlerTestsCommon : LightningTestSuite() {
 
         val result1 = paymentHandler.process(payToOpenRequest1, TestConstants.defaultBlockHeight)
         assertIs<IncomingPaymentHandler.ProcessAddResult.Pending>(result1)
+        assertEquals(emptyList(), result1.actions)
         val result2 = paymentHandler.process(payToOpenRequest2, TestConstants.defaultBlockHeight)
         assertIs<IncomingPaymentHandler.ProcessAddResult.Accepted>(result2)
+        val payToOpenResponse = PayToOpenResponseCommand(PayToOpenResponse(payToOpenRequest1.chainHash, payToOpenRequest1.paymentHash, PayToOpenResponse.Result.Success(incomingPayment.preimage)))
+        assertEquals(listOf(payToOpenResponse, payToOpenResponse), result2.actions)
```

Comment:
This is unrelated to this PR (probably for a follow-up): do we really want to send duplicate `payToOpenResponse`s? It would be trivial to make sure we send only one?

Reply:
No we don't, and we should also stop sending legacy `PayToOpenRequest` while we're at it